apps/air-discount-scheme/backend/src/app/modules/discount/discount.model.ts

```diff
@@ -26,8 +28,9 @@ export class Discount implements TDiscount {
   @ApiProperty()
   discountCode: string
 
-  @ApiProperty()
+  @ApiProperty({ type: [GQLConnectionDiscountCode] })
```

Comment:
Should this be `GQLConnectionDiscountCode[]`?

Reply:
If you look at some decorators the type attribute expects things in this fashion, f.x: 
![image](https://user-images.githubusercontent.com/77672665/202686494-f85fe30b-ab83-4d3a-b8f8-b825ebb7e3b8.png)

I did do GQLConnectionDiscountCode[] at first but the schema generated didn't indicate an array